radix-engine/src/blueprints/transaction_processor/tx_processor.rs

```diff
@@ -167,17 +170,27 @@ impl TransactionProcessorBlueprint {
             }
             match result.state {
                 TransactionProcessorState::YieldToChild(hash, value) => {
-                    received_value = Some(value);
-                    todo!()
+                    received_value = Some(value.clone());
+                    cur_thread = hash;
+
+                    // Context switch
+                    api.send_to_stack(cur_thread, value)?;
+                    api.switch_stack(cur_thread)?;
                 }
                 TransactionProcessorState::YieldToParent(value) => {
-                    received_value = Some(value);
-                    todo!()
+                    received_value = Some(value.clone());
+                    cur_thread = parent.unwrap();
+
+                    // Context switch
+                    api.send_to_stack(cur_thread, value)?;
+                    api.switch_stack(cur_thread)?;
```

Comment:
I think I'm missing it, but there doesn't seem to be any switching of execution to the target; nor any pausing/resumption of execution of the transaction processor.

This seems like the meat of the PoC to me